docs/content/src/learn/tutorials/saas-integrations/amazons3/amazon-s3-bucket-service/README.md

```diff
@@ -0,0 +1,138 @@
+# Amazon S3 Bucket Service
+
+## About
+
+Ballerina is an open-source programming language that supports developers to integrate their system easily with the 
+support of connectors. In this guide, we are mainly focusing on connecting to the Amazon Simple Storage Service API to create, store,download, and use data with other services.  
+
+The `wso2/amazons3` module allows you to perform the following operations.
+
+This example explains how to use the S3 client to connect with the Amazon S3 instance and to create a Amazon S3 bucket.
+
+You can find other integrations modules from [wso2-ballerina](https://github.com/wso2-ballerina) Github organization.
+
+## What you'll build
+
+This example explains how to use the Amazon S3 connector to connect with the Amazon S3 API and creating a new bucket on Amazon S3 instance with the provided name.
+
+![exposing Amazon S3 as a service](../../../../../assets/img/amazon-s3-bucket-service)
+
+<!-- INCLUDE_MD: ../../../../../tutorial-prerequisites.md -->
+
+<!-- INCLUDE_MD: ../../../../../tutorial-get-the-code.md -->
+
+## Implementation
+
+A Ballerina project is created for the integration use case explained above. Please follow the steps given 
+below to create the project and modules. You can learn about the Ballerina project and modules in this 
+[guide](https://ei.docs.wso2.com/en/latest/ballerina-integrator/develop/using-modules/#creating-a-project).
```

Comment:
Let's add a relative link here. Should be something like ../../../../../develop/managing-ballerina-code/

This is because this file will change based on https://github.com/wso2/ballerina-integrator/pull/457

Reply:
Fixed